src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -383,25 +381,14 @@ private void ProcessTraceroute(string targetNameOrAddress)
 #endif
                 var hopAddressString = discoveryReply.Address.ToString();
 
+                InitProcessPing(hopAddressString, out string routerName, out _);
```

Comment:
We should still put this in try-catch.

Reply:
You mean have an if/else? The method should be catching all errors here.

I'll add an if statement so we properly set some sort of router name based on `hopAddressString` if the name can't be resolved, etc.

Reply:
No, the method throws if router name can not be resolved
```suggestion
                try
                {
                    InitProcessPing(hopAddressString, out string routerName, out _);
                }
                catch
                {
                }
```

Reply:
The method writes an error, but it does not throw.

https://github.com/PowerShell/PowerShell/blob/f5ab069257c1e41ffc294ebb3676ae5c8830eb96/src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs#L637-L730

Reply:
I tested with artifact and get throw on first hop where router hasn't name. I guess it was in GetHostEntry().

Reply:
I still see the issue in latest artifact
![image](https://user-images.githubusercontent.com/22290914/71059225-21853880-2184-11ea-91ff-c7935c2ea72a.png)

Reply:
Huh! `GetHostEntry` itself throws? Thanks for following up on that. Yeah, I can sort that out. 🙂